src/opm/parser/eclipse/EclipseState/InitConfig/InitConfig.cpp

```diff
@@ -101,7 +101,8 @@ namespace Opm {
     }
 
     bool InitConfig::hasFoamConfig() const {
-        return !this->foamconfig.empty();
```

Comment:
Ehhh - method which unconditionally returns `true`can maybe be removed - or do you have future plans for this?

Reply:
Yes, that is why I kept it. It is not clear right now how we should approach the two distinct "gas mobility reduction models", a functional and a tabular one, they are given by different keywords. We currently support the tabular one. At some point I wanted data for both to be held by the FoamConfig, but the tabular data in FOAMMOB and FOAMADS fits better with the TableManager etc. approach. So until the functional variant is done (no plans to do it right now, the biggest change is not here but in the simulator code as then the water phase must be the transport phase not the gas phase), I do not really know how I want it to be.